homeassistant/components/cloud/tts.py

```diff
@@ -237,6 +222,7 @@ def supported_languages(self) -> list[str]:
     @property
     def supported_options(self) -> list[str]:
         """Return list of supported options like voice, emotion."""
+        # The gender option is deprecated and will be removed in 2024.9.0.
```

Comment:
The issue says October
```suggestion
        # The gender option is deprecated and will be removed in 2024.10.0.
```